src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/order/IsDone.java

```diff
@@ -0,0 +1,85 @@
+package ay2122s1_cs2103t_w16_2.btbb.model.order;
+
+import static ay2122s1_cs2103t_w16_2.btbb.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the done status of an order in BTBB.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class IsDone {
```

Comment:
I think this class name is very werid. Can it be something like `OrderCompletionStatus`